pkgs/by-name/sw/swhkd/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, rustPlatform
+, fetchurl
+, fetchFromGitHub
+, pkg-config
+, scdoc
+, libgcc
+, systemd
+}:
+rustPlatform.buildRustPackage rec {
+    pname = "swhkd";
+    version = "1.2.1-unstable-2024-04-06";
+
+    # split-output derivation, since there's a fair amount of associated data for
+    # pkexec and such.
+    outputs = [ "bin" "man" "out" ];
+
+    src = fetchFromGitHub {
+        owner = "waycrate";
+        repo = "swhkd";
+        # build from master, since the 1.2.1 makefile is unsutible for packaging
+        rev = "f8519a54900d72492a6c036b32e472c108d44dbf";
+        hash = "sha256-zyGyZOG8gAtsRkzSRH1M777fPv1wudbVsBrSTJ5CBnY=";
+      };
```

Comment:
```suggestion
      owner = "waycrate";
      repo = "swhkd";
      # build from master, since the 1.2.1 makefile is unsutible for packaging
      rev = "f8519a54900d72492a6c036b32e472c108d44dbf";
      hash = "sha256-zyGyZOG8gAtsRkzSRH1M777fPv1wudbVsBrSTJ5CBnY=";
    };
```